packages/components/src/components/typography/Text/Text.stories.tsx

```diff
@@ -24,38 +24,38 @@ const meta: Meta = {
 } as Meta;
 export default meta;
 
-export const Text: StoryObj = {
-    render: props => (
+export const Text: StoryObj<typeof TextComponent> = {
+    render: ({ variant, color, ...rest }) => (
```

Comment:
We shouldn't pass variant and color because type excludes these two props together. I'm surprised it worked before, now CI is failing